addons/product/models/product_template.py

```diff
@@ -500,6 +500,9 @@ def _get_related_fields_variant_template(self):
     @api.model_create_multi
     def create(self, vals_list):
         ''' Store the initial standard price in order to be able to retrieve the cost of a product template for a given date'''
+        for vals in vals_list:
+            if 'uom_id' in vals and not 'uom_po_id' in vals:
```

Comment:
```suggestion
            if 'uom_id' in vals and 'uom_po_id' not in vals:
```
`X not in Y` is generally preferable over `not X in Y`